### ISSUE TITLE
App crashes when trying to join a meet

### ISSUE THREAD

**Issue by EmanuelBalaban** [jitsi/jitsi-meet-flutter-sdk#28]
### Description:

The app crashes when trying to join a meet using `jitsi-meet-flutter-sdk` pub.

### Steps to reproduce:

1. Create a new flutter project
2. Do all the setup steps
3. Create a sample page
4. Try to join a meet

### **StackTrace**

<details>
  <summary>Click to expand</summary>

I/flutter ( 8238): MethodResponse{isSuccess: true, message: Successfully joined meeting EmanuelTestHello, error: null}
D/OrientationModule( 8238): onActivityPaused
D/AppCompatDelegate( 8238): Checking for metadata for AppLocalesMetadataHolderService : Service not found
D/OrientationModule( 8238): onActivityCreated
W/_store_frontend( 8238): Accessing hidden method Landroid/view/View;->computeFitSystemWindows(Landroid/graphics/Rect;Landroid/graphics/Rect;)Z (unsupported, reflection, allowed)
W/_store_frontend( 8238): Accessing hidden method Landroid/view/ViewGroup;->makeOptionalFitsSystemWindows()V (unsupported, reflection, allowed)
I/org.webrtc.Logging( 8238): EglBase14Impl: Using OpenGL ES version 2
D/ReactInstanceManagerHolder( 8238): initializing RN with Activity
W/_store_frontend( 8238): Accessing hidden method Landroid/content/Context;->destroy()V (blocked, linking, denied)
W/_store_frontend( 8238): Accessing hidden method Landroid/content/Context;->destroy()V (blocked, linking, denied)
D/AndroidRuntime( 8238): Shutting down VM
E/AndroidRuntime( 8238): FATAL EXCEPTION: main
E/AndroidRuntime( 8238): Process: com.emanuel.neat_store_frontend, PID: 8238
E/AndroidRuntime( 8238): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.emanuel.neat_store_frontend/org.jitsi.jitsi_meet_flutter_sdk.WrapperJitsiMeetActivity}: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/AndroidRuntime( 8238): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3782)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
E/AndroidRuntime( 8238): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
E/AndroidRuntime( 8238): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
E/AndroidRuntime( 8238): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
E/AndroidRuntime( 8238): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/AndroidRuntime( 8238): 	at android.os.Looper.loopOnce(Looper.java:205)
E/AndroidRuntime( 8238): 	at android.os.Looper.loop(Looper.java:294)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/AndroidRuntime( 8238): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 8238): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/AndroidRuntime( 8238): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/AndroidRuntime( 8238): Caused by: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/AndroidRuntime( 8238): Caused by: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/AndroidRuntime( 8238): Caused by: java.lang.reflect.InvocationTargetException
E/AndroidRuntime( 8238): 	at java.lang.reflect.Constructor.newInstance0(Native Method)
E/AndroidRuntime( 8238): 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.createView(LayoutInflater.java:866)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1018)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:973)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:694)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
E/AndroidRuntime( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:485)
E/AndroidRuntime( 8238): 	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:775)
E/AndroidRuntime( 8238): 	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:197)
E/AndroidRuntime( 8238): 	at org.jitsi.meet.sdk.JitsiMeetActivity.onCreate(JitsiMeetActivity.java:103)
E/AndroidRuntime( 8238): 	at org.jitsi.jitsi_meet_flutter_sdk.WrapperJitsiMeetActivity.onCreate(WrapperJitsiMeetActivity.kt:35)
E/AndroidRuntime( 8238): 	at android.app.Activity.performCreate(Activity.java:8595)
E/AndroidRuntime( 8238): 	at android.app.Activity.performCreate(Activity.java:8573)
E/AndroidRuntime( 8238): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3764)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
E/AndroidRuntime( 8238): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
E/AndroidRuntime( 8238): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
E/AndroidRuntime( 8238): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
E/AndroidRuntime( 8238): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/AndroidRuntime( 8238): 	at android.os.Looper.loopOnce(Looper.java:205)
E/AndroidRuntime( 8238): 	at android.os.Looper.loop(Looper.java:294)
E/AndroidRuntime( 8238): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/AndroidRuntime( 8238): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime( 8238): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/AndroidRuntime( 8238): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/AndroidRuntime( 8238): Caused by: java.lang.VerifyError: Verifier rejected class com.oblador.performance.PerformanceModule: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMark) failed to verify: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMark): [0x27] register v1 has type Reference: com.facebook.react.bridge.WritableMap but expected Reference: com.facebook.react.bridge.ReadableMap
E/AndroidRuntime( 8238):  void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMetric) failed to verify: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMetric): [0x30] register v1 has type Reference: com.facebook.react.bridge.WritableMap but expected Reference: com.facebook.react.bridge.ReadableMap (declaration of 'com.oblador.performance.PerformanceModule' appears in /data/app/~~FVuexvyOTjyEcG0GsyBX3Q==/com.emanuel.neat_store_frontend-ZRwnlJsJfvujfJ4lMijOSw==/base.apk!classes19.dex)
E/AndroidRuntime( 8238): 	at com.oblador.performance.PerformancePackage.<init>(PerformancePackage.java:19)
E/AndroidRuntime( 8238): 	at org.jitsi.meet.sdk.ReactInstanceManagerHolder.getReactNativePackages(ReactInstanceManagerHolder.java:89)
E/AndroidRuntime( 8238): 	at org.jitsi.meet.sdk.ReactInstanceManagerHolder.initReactInstanceManager(ReactInstanceManagerHolder.java:253)
E/AndroidRuntime( 8238): 	at org.jitsi.meet.sdk.JitsiMeetView.initialize(JitsiMeetView.java:201)
E/AndroidRuntime( 8238): 	at org.jitsi.meet.sdk.JitsiMeetView.<init>(JitsiMeetView.java:102)
E/AndroidRuntime( 8238): 	... 30 more
E/JitsiMeetSDK( 8238): JitsiMeetUncaughtExceptionHandler FATAL ERROR
E/JitsiMeetSDK( 8238): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.emanuel.neat_store_frontend/org.jitsi.jitsi_meet_flutter_sdk.WrapperJitsiMeetActivity}: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3782)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
E/JitsiMeetSDK( 8238): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loopOnce(Looper.java:205)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loop(Looper.java:294)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Method.invoke(Native Method)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/JitsiMeetSDK( 8238): Caused by: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/JitsiMeetSDK( 8238): Caused by: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/JitsiMeetSDK( 8238): Caused by: java.lang.reflect.InvocationTargetException
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Constructor.newInstance0(Native Method)
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.createView(LayoutInflater.java:866)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1018)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:973)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:694)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:485)
E/JitsiMeetSDK( 8238): 	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:775)
E/JitsiMeetSDK( 8238): 	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:197)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.JitsiMeetActivity.onCreate(JitsiMeetActivity.java:103)
E/JitsiMeetSDK( 8238): 	at org.jitsi.jitsi_meet_flutter_sdk.WrapperJitsiMeetActivity.onCreate(WrapperJitsiMeetActivity.kt:35)
E/JitsiMeetSDK( 8238): 	at android.app.Activity.performCreate(Activity.java:8595)
E/JitsiMeetSDK( 8238): 	at android.app.Activity.performCreate(Activity.java:8573)
E/JitsiMeetSDK( 8238): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3764)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
E/JitsiMeetSDK( 8238): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loopOnce(Looper.java:205)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loop(Looper.java:294)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Method.invoke(Native Method)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/JitsiMeetSDK( 8238): Caused by: java.lang.VerifyError: Verifier rejected class com.oblador.performance.PerformanceModule: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMark) failed to verify: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMark): [0x27] register v1 has type Reference: com.facebook.react.bridge.WritableMap but expected Reference: com.facebook.react.bridge.ReadableMap
E/JitsiMeetSDK( 8238):  void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMetric) failed to verify: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMetric): [0x30] register v1 has type Reference: com.facebook.react.bridge.WritableMap but expected Reference: com.facebook.react.bridge.ReadableMap (declaration of 'com.oblador.performance.PerformanceModule' appears in /data/app/~~FVuexvyOTjyEcG0GsyBX3Q==/com.emanuel.neat_store_frontend-ZRwnlJsJfvujfJ4lMijOSw==/base.apk!classes19.dex)
E/JitsiMeetSDK( 8238): 	at com.oblador.performance.PerformancePackage.<init>(PerformancePackage.java:19)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.ReactInstanceManagerHolder.getReactNativePackages(ReactInstanceManagerHolder.java:89)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.ReactInstanceManagerHolder.initReactInstanceManager(ReactInstanceManagerHolder.java:253)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.JitsiMeetView.initialize(JitsiMeetView.java:201)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.JitsiMeetView.<init>(JitsiMeetView.java:102)
E/JitsiMeetSDK( 8238): 	... 30 more
E/JitsiMeetSDK( 8238): 
E/JitsiMeetSDK( 8238): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.emanuel.neat_store_frontend/org.jitsi.jitsi_meet_flutter_sdk.WrapperJitsiMeetActivity}: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3782)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
E/JitsiMeetSDK( 8238): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loopOnce(Looper.java:205)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loop(Looper.java:294)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Method.invoke(Native Method)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/JitsiMeetSDK( 8238): Caused by: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/JitsiMeetSDK( 8238): Caused by: android.view.InflateException: Binary XML file line jitsi/jitsi-meet#12 in com.emanuel.neat_store_frontend:layout/activity_jitsi_meet: Error inflating class org.jitsi.meet.sdk.JitsiMeetView
E/JitsiMeetSDK( 8238): Caused by: java.lang.reflect.InvocationTargetException
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Constructor.newInstance0(Native Method)
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.createView(LayoutInflater.java:866)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1018)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:973)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:694)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
E/JitsiMeetSDK( 8238): 	at android.view.LayoutInflater.inflate(LayoutInflater.java:485)
E/JitsiMeetSDK( 8238): 	at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:775)
E/JitsiMeetSDK( 8238): 	at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:197)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.JitsiMeetActivity.onCreate(JitsiMeetActivity.java:103)
E/JitsiMeetSDK( 8238): 	at org.jitsi.jitsi_meet_flutter_sdk.WrapperJitsiMeetActivity.onCreate(WrapperJitsiMeetActivity.kt:35)
E/JitsiMeetSDK( 8238): 	at android.app.Activity.performCreate(Activity.java:8595)
E/JitsiMeetSDK( 8238): 	at android.app.Activity.performCreate(Activity.java:8573)
E/JitsiMeetSDK( 8238): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3764)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3922)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
E/JitsiMeetSDK( 8238): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2443)
E/JitsiMeetSDK( 8238): 	at android.os.Handler.dispatchMessage(Handler.java:106)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loopOnce(Looper.java:205)
E/JitsiMeetSDK( 8238): 	at android.os.Looper.loop(Looper.java:294)
E/JitsiMeetSDK( 8238): 	at android.app.ActivityThread.main(ActivityThread.java:8177)
E/JitsiMeetSDK( 8238): 	at java.lang.reflect.Method.invoke(Native Method)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
E/JitsiMeetSDK( 8238): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
E/JitsiMeetSDK( 8238): Caused by: java.lang.VerifyError: Verifier rejected class com.oblador.performance.PerformanceModule: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMark) failed to verify: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMark): [0x27] register v1 has type Reference: com.facebook.react.bridge.WritableMap but expected Reference: com.facebook.react.bridge.ReadableMap
E/JitsiMeetSDK( 8238):  void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMetric) failed to verify: void com.oblador.performance.PerformanceModule.emit(com.oblador.performance.PerformanceMetric): [0x30] register v1 has type Reference: com.facebook.react.bridge.WritableMap but expected Reference: com.facebook.react.bridge.ReadableMap (declaration of 'com.oblador.performance.PerformanceModule' appears in /data/app/~~FVuexvyOTjyEcG0GsyBX3Q==/com.emanuel.neat_store_frontend-ZRwnlJsJfvujfJ4lMijOSw==/base.apk!classes19.dex)
E/JitsiMeetSDK( 8238): 	at com.oblador.performance.PerformancePackage.<init>(PerformancePackage.java:19)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.ReactInstanceManagerHolder.getReactNativePackages(ReactInstanceManagerHolder.java:89)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.ReactInstanceManagerHolder.initReactInstanceManager(ReactInstanceManagerHolder.java:253)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.JitsiMeetView.initialize(JitsiMeetView.java:201)
E/JitsiMeetSDK( 8238): 	at org.jitsi.meet.sdk.JitsiMeetView.<init>(JitsiMeetView.java:102)
E/JitsiMeetSDK( 8238): 	... 30 more
I/_store_frontend( 8238): Thread[5,tid=8243,WaitingInMainSignalCatcherLoop,Thread*=0x71ca221676f0,peer=0x12f00238,"Signal Catcher"]: reacting to signal 3
I/_store_frontend( 8238): 
I/_store_frontend( 8238): Wrote stack traces to tombstoned
Lost connection to device.
</details>

### Code

<details>
  <summary>Click to expand</summary>
  
```dart
import 'package:auto_route/auto_route.dart';
import 'package:flutter/material.dart';
import 'package:jitsi_meet_flutter_sdk/jitsi_meet_flutter_sdk.dart';

@RoutePage()
class MeetPage extends StatefulWidget {
  const MeetPage({super.key});

  @override
  State<MeetPage> createState() => _MeetPageState();
}

class _MeetPageState extends State<MeetPage> {
  final jitsiMeet = JitsiMeet();

  @override
  Widget build(BuildContext context) {
    return Center(
      child: ElevatedButton(
        onPressed: () async {
          final response = await jitsiMeet.join(
            JitsiMeetConferenceOptions(
              serverURL: 'https://meet.jit.si',
              room: 'EmanuelTestHello',
              userInfo: JitsiMeetUserInfo(
                displayName: 'Emanuel Balaban',
                email: 'emanuelbalaban@gmail.com',
              ),
            ),
            JitsiMeetEventListener(
              conferenceWillJoin: (url) {
                print('conferenceWillJoin $url');
              },
              conferenceJoined: (url) {
                print('conferenceJoined $url');
              },
              conferenceTerminated: (url, error) {
                print('conferenceTerminated $url $error');
              },
              participantJoined: (email, name, role, participantId) {
                print(
                  'participantJoined '
                  'email: $email, '
                  'name: $name, '
                  'role: $role, '
                  'participantId: $participantId',
                );
              },
              readyToClose: () {
                print('readyToClose');
              },
            ),
          );

          print(response);
        },
        child: Text('Click me'),
      ),
    );
  }
}
```
</details>

**Comment by mohamedsoliman121** [comment]
are you fix it brother ?

**Comment by ArunKarthikAnandKumar** [comment]
can anyone look into this issue and fix it as it is a bit urgent

**Comment by saghul** [comment]
What version of the Gradle plugin are you using?

**Comment by ArunKarthikAnandKumar** [comment]
7.2.2

**Comment by Holofox** [comment]
@saghul, when I updated the flutter package version 0.1.8/0.1.9 -> 0.2.0 (JitsiMeetSDK 8.5.0/8.60 -> 9.0.0), the iOS application started crashing after joining the meet. It crashes only on iOS and only on this version of the package.

[logs.txt](https://github.com/jitsi/jitsi-meet-flutter-sdk/files/14598032/logs.txt)

**Comment by saghul** [comment]
Do you also observe it on a real device?

**Comment by saghul** [comment]
Hols on, we'll need to do a .1 release since the 9.0.0 iOS SDK is busted.

Ping @Calinteodor

**Comment by Holofox** [comment]
@saghul, on a real device when the app starts I get an unhandled exception:
```
dyld[36593]: Symbol not found: _$s10GiphyUISDK0A14ViewControllerC19selectedContentTypeAA010GPHContentG0OvsTj
  Referenced from: <893FD909-BD9F-3766-91CA-E8DA130B2B82> /private/var/containers/Bundle/Application/185B2484-994B-4536-A9FF-4B6158F7694F/Runner.app/Frameworks/JitsiMeetSDK.framework/JitsiMeetSDK
  Expected in:     <45CBFAF2-01DD-37E5-8BD7-68C0D388191E> /private/var/containers/Bundle/Application/185B2484-994B-4536-A9FF-4B6158F7694F/Runner.app/Frameworks/GiphyUISDK.framework/GiphyUISDK
```

**Comment by saghul** [comment]
Ah right. That got fixed in our iOS SDK 9.0.1.

We'll make a new flutter SDK defending on it.

**Comment by Calinteodor** [comment]
> Ah right. That got fixed in our iOS SDK 9.0.1.
> 
> We'll make a new flutter SDK defending on it.

Please update to 9.0.2. Thank you!
